test/Field/Raw/Textual.hs

```diff
@@ -0,0 +1,49 @@
+{-| A simple set of tests for the raw parser
+-}
+
+module Field.Raw.Textual
+  ( gen_test_parsing
+  ) where
+
+import Data.List (sort)
+import TinyLang.Prelude
+import TinyLang.Field.Raw.Parser
+import Test.Tasty
+import Test.Tasty.Golden
+import Test.Tasty.HUnit
+
+import System.FilePath
+import System.FilePath.Glob
+
+testDir :: FilePath
+testDir = "test" </> "Field" </> "Raw" </> "golden"
+
+gen_test_parsing :: IO TestTree
+gen_test_parsing =
+    do
+        files <- testFiles
```

Comment:
So annoying that you can't do `IO` in `TestTree`.

Reply:
Yes.  I am afraid that was intentional:
https://stackoverflow.com/a/33046723